### ISSUE TITLE
"No beatmaps available" shouldn't always show when no beatmap is currently selected

### ISSUE THREAD

**Issue by jojothka123** [ppy/osu#17666]
### Type

Cosmetic

### Bug description

music playback overlay displays inaccurate text when in a multiplayer lobby without having downloaded the map

### Screenshots or videos

![osu_2022-04-05_19-57-03](https://user-images.githubusercontent.com/103065432/161819779-8dc6087a-dc8f-4b35-a4f1-0329c174142d.jpg)

### Version

2022.405.0-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/8420742/network.log)
[performance.log](https://github.com/ppy/osu/files/8420743/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8420744/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8420745/updater.log)
[database.log](https://github.com/ppy/osu/files/8420746/database.log)

**Comment by KantoChampion** [comment]
So in the event a beatmap is not selected, this message is displayed? In other words, beatmaps are available and the message should prompt the user to instead, "select a beatmap"?

**Comment by peppy** [comment]
What do you want it to display in this case? Please provide a suggestion.

**Comment by jojothka123** [comment]
> 

No, the message is displayed when the selected beatmap has not yet been loaded. I'd suggest either continuing to play the previously playing song or changing the text to "selected beatmap not loaded! / please load the selected beatmap!".
